test/configCases/parsing/harmony-reexport/webpack.config.js

```diff
@@ -0,0 +1,2 @@
+/** @type {import("../../../../").Configuration} */
+module.exports = {};
```

Comment:
When this doesn't need special config, we should move it into `test/cases` so it's tested with different configs.

Maybe wrapping some assertions with `if(process.env.NODE_ENV === "production")` (e. g. for optimizations that only run in production mode)

Reply:
this was done intentionally to run 3 times (check for cache invalidation)

Reply:
Ah yes that makes sense